drizzlepac/haputils/catalog_utils.py

```diff
@@ -2004,6 +2001,7 @@ def measure_sources(self, filter_name):
             log.info("image name: {}".format(self.imgname))
             log.info("Generating empty segment catalog.")
             log.info("")
+            self._define_empty_table(None)
 
```

Comment:
Maybe I am tired today.  I am wondering why _define_empty_table has a None being passed.  Is _define_empty_table(None) the same as _define_empty_table()?

Reply:
The function `_define_empty_table()` needs a parameter to be passed in as that is used to populate the `.sources` attribute for the catalog class.  When called in `measure_sources`, there are no sources and the seg map was returned as None by `detect_sources`, the function was simply passed a `None` instead of the `segm_img` attribute which had a value of `None`.